### PR TITLE
Add documentation of new environment variables

### PR DESCRIPTION
**Pull request by billschereriii**
Update documentation to detail new environment variables.

### CODE REVIEW

File: doc/runtime.rst [hunk 1]
```diff
@@ -107,6 +107,15 @@ The functions for changing this default behavior are:
     The function ``Client.use_model_ensemble_prefix()`` controls
     object prefixing for model and script data.
 
+Model Execution Environment Variable
+====================================
+
+SmartRedis enables a timeout on the length of time that is allowed
+to elapse while waiting for a model to elapse. This is controlled
```
Comment (mellis13) [inline doc/runtime.rst:114]: should this be "waiting for a model to execute."?.

**Review by mellis13 — APPROVED** [review]
Thanks for the update!  Feel free to merge after group discussion on defaults.

File: doc/runtime.rst [hunk 2]
```diff
@@ -107,6 +107,15 @@ The functions for changing this default behavior are:
     The function ``Client.use_model_ensemble_prefix()`` controls
     object prefixing for model and script data.
 
+Model Execution Environment Variable
+====================================
+
+SmartRedis enables a timeout on the length of time that is allowed
```
Comment (ashao) [inline doc/runtime.rst:113]: I might suggest leading with the variable name, i.e. `SR_MODEL_TIMEOUT` sets the amount of time that the client will wait for a model to execute [...]

File: doc/runtime.rst [hunk 3]
```diff
@@ -130,3 +139,11 @@ The user can set these environment variables to adjust command execution behavio
 ``SR_CMD_TIMEOUT`` should be specified in seconds.  Note that ``SR_CMD_INTERVAL``
 and ``SR_CMD_TIMEOUT`` are read during client initialization and not
```
Comment (ashao) [inline doc/runtime.rst:140]: Isn't this behavior true of all of the environment variables for the client?

**Review by ashao — CHANGES_REQUESTED** [review]
Very minor comments for clarity. Thank you @billschereriii for making sure these were documented.

File: doc/runtime.rst (hunk 3 above)
Comment (billschereriii) [reply doc/runtime.rst:139]: We don't currently have an overview of environment variables, but maybe we should add a table to the top of this text that has overview info like this. Not something that can fit into this cycle though

**Review by ashao — APPROVED** [review]

**Pull request merged**